app/gui/src/dashboard/services/remoteBackendPaths.ts

```diff
@@ -219,6 +221,10 @@ export function deleteUserGroupPath(groupId: backend.UserGroupId) {
 export function getCheckoutSessionPath(checkoutSessionId: backend.CheckoutSessionId) {
   return `${GET_CHECKOUT_SESSION_PATH}/${checkoutSessionId}`
 }
+/** Relative HTTP path to the "get oauth callback" endpoint of the Cloud backend API. */
+export function getOauthCallbackPath(service: backend.CredentialInput['type']) {
+  return `libs/${service}/oauth/callback`
```

Comment:
I thought we were aiming for `oauth/${service}/callback`?

I really don't see why there's libs in there - @radeusgd from libs was working on the endpoint, yes, but other than that it is not closer tied to libs than e.g. secrets or datalinks are, and these don't have such a prefix.